tools/mo/openvino/tools/mo/front/onnx/maxunpoolONNX_to_maxunpool.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+from openvino.tools.mo.front.common.replacement import FrontReplacementSubgraph
+from openvino.tools.mo.graph.graph import Graph
+from openvino.tools.mo.ops.scatter import ScatterElementsUpdate
+from openvino.tools.mo.ops.const import Const
+from openvino.tools.mo.ops.elementwise import Mul
+from openvino.tools.mo.ops.reshape import Reshape
+from openvino.tools.mo.ops.shape import Shape
+
+
+class MaxUnpoolFrontReplacer(FrontReplacementSubgraph):
+    enabled = True
+
+    def pattern(self):
+        return dict(
+            nodes=[
+                ("max_pool0", dict(op="MaxPool")),
+                ("max_pool1", dict(op="MaxPool")),
+                ("slice", dict(op="AttributedSlice")),
+                ("sub", dict(op="Sub")),
+                ("unpool", dict(op="max_unpool2d")),
```

Comment:
is `max_unpool2d` PyTorch operation that is a custom for ONNX opset?

Reply:
@likholat, can you please add a full example with PyTorch model in PR description?